apis/apps/v1alpha1/nebulacluster_common.go

```diff
@@ -370,14 +374,76 @@ func generateAgentContainer(c NebulaClusterComponent, init bool) corev1.Containe
 	return container
 }
 
+func genNodeLabelsContainer(nc *NebulaCluster) corev1.Container {
```

Comment:
Maybe `kubectl get node node1 -o jsonpath='{.metadata.labels.topology\.kubernetes\.io/zone}'` can meet the needs, and there is no need for a new image.